### DISCUSSION TITLE
vue2的项目可以引入vue3的公共组件库吗？

### DISCUSSION THREAD

**Discussion by lxmbetterman** [element-plus/element-plus#2630] [Q&A] [answered]
想请教一个问题：
1.如果我当前项目是基于vue2的，我引入element-plus（vue3），能正常工作吗？
2.相反，如果我基于vue-cli4搭建vue3的项目，我能否引入vue2的公共ui组件库呢？

能否普及一下，vue2/vue3 公共组件库相互间引入，会存在什么样的问题

**Comment by iamkun** [comment, marked as answer]
No, you can't. 

There's a lot of breaking changes between Vue 2 and 3
